website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,137 @@
+</Subtitle> 
+
+# Installation
+
+```
+yarn add @commercetools-frontend/mc-scripts
+
+# or
+
+npm --save install @commercetools-frontend/mc-scripts
+```
+
+# Usage
+To create a custom webpack configuration for a Merchant Center application, you can use `createWebpackConfigForProduction` for production and `createWebpackConfigForDevelopment` for development.
+These are factory functions used to create a default webpack config. They both accept some optional arguments.
+
+## Production
+For production mode, you should create a `webpack.config.prod.(cjs, js)` file and use the function `createWebpackConfigForProduction` to create a default webpack config.
+
+```tsx title="webpack.config.prod.cjs"
+const { createWebpackConfigForProduction } = require("@commercetools-frontend/mc-scripts");
+
+const config = createWebpackConfigForProduction(options: TWebpackConfigOptions);
+
+// you can include your webpack config by updating the default config object
+config.devtool: 'cheap-source-map'
+
+module.exports = config; 
+```
+
+### options
+```ts
+type TWebpackConfigOptions = {
+  /**
+   * The absolute path to the application entry point file.
+   */
+  entryPoint?: string;
+  /**
+   * A list of folders where Webpack should look for source files.
+   */
+  sourceFolders?: string[];
+  /**
+   * Options related to Postcss plugins. See `createPostcssConfig` function.
+   */
+  postcssOptions?: TPostcssConfigOptions;
+  /**
+   * Options to enable/disable certain functionalities of the Webpack config.
+   */
+  toggleFlags?: TWebpackConfigToggleFlagsForProduction
+}
+
+type TWebpackConfigToggleFlagsForProduction = {
+  /**
+   * Allow to disable index.html generation in case it's not necessary (e.g. for Storybook)
+   */
+  generateIndexHtml?: boolean;
+  /**
+   * Some environments do not require `core-js` and can hence disable
+   * it explicitly. This will disable `core-js` for `preset-env` and the
+   * `plugin-transform-runtime`.
+   */
+  disableCoreJs?: boolean;
+  /**
+    * Allow to disable CSS extraction in case it's not necessary (e.g. for Storybook)
+    */
+  enableExtractCss?: boolean;
+  /**
+    * Some plugins spawn workers to speed up the build. However this can cause trouble on
+    * certain machines local and CI. This flag set to limit or disable any parallelism.
+    * Options:
+    *    `true` to default to the machines number of CPUs
+    *    `false` to disable any parallelism
+    *    `int` for a specific number of CPUs
+    */
+  parallelism: true;
+}
+
+type TPostcssConfigOptions = {
+  // A list of paths where to look for files used by the `@import` statements.
+  postcssImportPaths?: string[];
+  // A list of paths where to look for files with custom media queries.
+  postcssCustomMediaPaths?: string[];
+  // A list of paths where to look for files with custom properties.
+  postcssCustomPropertiesPaths?: string[];
+}
+```
+
+
+## For Development
```

Comment:
I would move this before the "For production" section.

Also see the other suggestions that should be applied here as well.